README.md

```diff
@@ -4,7 +4,7 @@
 This is a Scala/Play Framework microservice that provides a UI to search a database of MTD VAT Software Providers. 
 Having suitable software to submit VAT returns is a requirement of MTD VAT.
 
-The list of software providers is loaded from the `SofwareProviders.json` file. To add new providers, update this file and the `TestSoftwareProviders.json` file then run the tests.
```

Comment:
revert this file

Reply:
there was a typo, i corrected it. why are reverting it?

Reply:
Sorry, I did not notice, you changed the content. Thats good catch. Leave it as is. I thought its a space.